### DISCUSSION TITLE
part-ii/2016-119/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#273] [Question comments]
Part II 2016 Probability and Measure: Paper 2, Section II, J 

https://questions.tripos.org/part-ii/2016-119/

**Comment by eha1234** [comment]
Not 100% confident but anyways
<details><summary>(b)</summary>

Rescaling, wlog we're working in a probability space, and so $||f||_q \leq  ||f||_p$ for $q < p$. Hence $||f||_p$ is increasing in $p$. Moreover, $(\int |f|^p d\mu)^{1/p} \leq (\int_E M^p 1_A d\mu)^{1/p}$ for $M = ||f||_{\infty}$, $A$ such that $\mu(A) = 1$. 

Now fix $\epsilon > 0$ and let $B = \{ x: f(x) > M-\epsilon\}$. Have $\mu(B) > 0$.
Then $\int |f|^p d\mu \geq (M-\epsilon)^p 1_B d\mu$ and so $||f||_p \geq (M-\epsilon) (\mu(B))^{1/p}$. As $p \rightarrow \infty$, $(\mu(B) )^{1/p} \rightarrow 1$, so eventually $||f||_p \geq M - 2\epsilon$. Result follows. 
</details>

<details><summary>(c)</summary>

Holder on $|f|^{p/s} and |f|^{q/t}$ for $1/s+1/t = 1$
</details>
